src/topology/mod.rs

```diff
@@ -143,7 +143,7 @@ impl RunningTopology {
             let remaining_components = check_handles2.keys().cloned().collect::<Vec<_>>();
 
             error!(
-                "Failed to gracefully shut down in time. Killing: {}",
+                "Failed to gracefully shut down in time. Killing: {}.",
                 remaining_components.join(", ")
```

Comment:
Would this make sense?
```suggestion
                "Failed to gracefully shut down in time. Killing components.",
                components = %remaining_components
```